### ISSUE TITLE
debug sections stripped

### ISSUE THREAD

**Issue by pitust** [rust-osdev/bootimage#68]
The debug sections (eg. debug_line) are stripped even in debug builds, even though kernels might want to use that for things like line numbers.

**Comment by phil-opp** [comment]
Could you clarify what you're trying to do? Parse the kernel ELF executable from within the loaded kernel?

**Comment by pitust** [comment]
Yes, indeed. I tweaked the bootloader to give me the entire kernel ELF. Now i am trying to parse it for line numbers (using xmas_elf and gimli).

**Comment by phil-opp** [comment]
Interesting! I guess we could add a configuration option to not strip the ELF executable.
